domains/Accounts/Tests/Feature/UserCreateTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Domains\Accounts\Tests\Feature;
+
+use Domains\Accounts\Models\User;
+use Domains\Accounts\Notifications\VerifyEmailNotification;
+use Faker\Factory;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Notification;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class UserCreateTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected Generator $faker;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->faker = Factory::create();
+        Notification::fake();
+    }
+
+    /** @test */
+    public function it_fails_to_create_a_user_on_validation_errors(): void
+    {
+        $this->post(route('accounts.store'), [])
+            ->seeJsonStructure([
+                'name',
+                'email',
+                'password',
+            ])->assertResponseStatus(Response::HTTP_UNPROCESSABLE_ENTITY);
+
+        Notification::assertNothingSent();
+    }
+
+    /** @test */
+    public function it_creates_a_user_with_pending_email_verification(): void
+    {
+        $payload = [
+            'name' => $this->faker->name,
+            'email' => $this->faker->safeEmail,
+            'password' => $this->faker->password,
+        ];
+
+        $response = $this->post(route('accounts.store'), $payload);
+
+        self::assertTrue($response->response->isEmpty());
```

Comment:
Nit: we can avoid referencing the response property and call `isEmpty()` directly on the $response variable

Reply:
Only when using `$this->call`. When using `$this->post` it fails the test:
```
Time: 00:00.747, Memory: 22.00 MB

There was 1 failure:

1) Domains\Accounts\Tests\Feature\UserCreateTest::it_creates_a_user_with_pending_email_verification
Failed asserting that PHPUnit\Framework\Constraint\IsEmpty Object &0000000079b10064000000001c029e8f (
    'exporter' => null
) is true.
```

Reply:
Funny, I was pretty sure it'll work :man_shrugging: Thanks for clarifying.